python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -325,6 +325,15 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         doc="Apply gains to the image?",
         default=True,
     )
+    useGainsFrom = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Where to retrieve the gains.",
```

Comment:
Please comment in the doc string that this config is unused if `doBootstrap` is True.